includes/elements/container.php

```diff
@@ -416,16 +434,29 @@ protected function register_container_layout_controls() {
 			]
 		);
 
+		$placeholder_width = [];
+
+		if ( ! empty( $this->active_kit->get_settings_for_display( 'space_between_widgets' )['size'] ) ) {
+			$placeholder_width = [
+				'desktop' => [
+					'placeholder' => [
+						'size' => $this->active_kit->get_settings_for_display( 'space_between_widgets' )['size'],
+						'unit' => 'px',
+					],
+				],
+			];
+		}
+
 		$this->add_group_control(
 			Group_Control_Flex_Container::get_type(),
 			[
 				'name' => 'flex',
 				'selector' => '{{WRAPPER}}',
 				'fields_options' => [
 					'gap' => [
-						'label' => esc_html_x( 'Gap between elements', 'Flex Container Control', 'elementor' ),
```

Comment:
@hein-obox Please just double check this label :-) I might be wrong, but I think you are changing it back. I had to update it to `Gap between elements` in another task. Maybe just pull release/3.7.0 in here to be sure.

Reply:
@louiswol94 Fixed by merging